code/src/App.js

```diff
@@ -1,9 +1,91 @@
-import React from 'react'
+import React, { useState, useEffect} from 'react'
+
+import { API_URL, HEART_URL } from './reusables/urls'
+
+import Form from './components/Form'
+import MessageList from './components/MessageList'
+import Header from './components/Header'
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([])
+  const [messageNew, setMessageNew] = useState('')
+  // const [error, setError] = useState('')
+  const [characters, setCharacters] = useState(messageNew.length)
+
+  useEffect(() => {
+    fetchMessageList()
+  }, [])
+
+  const fetchMessageList = () => {
+      fetch(API_URL)
+      .then(res => res.json())
+      .then(messages => setMessageList(messages))
+      .catch(err => console.error(err))
```

Comment:
console outputs can't be present in the production code.

Reply:
@KarolCislowski I have removed this now!